letsencrypt/client/standalone_authenticator.py

```diff
@@ -0,0 +1,519 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
```

Comment:
Please remove sheebang, coding stanza (it is redundant here, isn't it?) and empty line.